gdk4/src/auto/toplevel_layout.rs

```diff
@@ -39,11 +40,19 @@ impl ToplevelLayout {
     }
 
     #[doc(alias = "gdk_toplevel_layout_get_fullscreen")]
-    pub fn get_fullscreen(&self) -> bool {
+    pub fn get_fullscreen(&self) -> Option<bool> {
```

Comment:
Hmm, what makes gir mis-regenerates this one? it should only return a bool per the docs https://developer.gnome.org/gdk4/stable/GdkToplevelLayout.html#gdk-toplevel-layout-get-fullscreen

Reply:
No, this is completely correct and the docs are outdated: https://gitlab.gnome.org/GNOME/gtk/-/blob/master/gdk/gdktoplevellayout.c#L261

Reply:
Ah yes my bad. I somehow missed it's an out param.